tests/test_plasmodium.py

```diff
@@ -186,7 +186,7 @@ def test_setup_returns_config_correctly(self):
         self.assertEqual(
             plasmodium.CONF,
             {
-                "default_url": "gs://test_plasmodium_release/",
+                "default_url": "s3://test_plasmodium_release/",
```

Comment:
This change ensures the tests don't try to access GCS within this test module (using a GCS URL triggers an attempt to authenticate, which then requires secrets, which then won't work on a PR from a fork).